Toolchain/BuildVcpkg.py

```diff
@@ -22,14 +30,24 @@ def main() -> int:
         bootstrapped_vcpkg_version = subprocess.check_output(
             ["git", "-C", vcpkg_checkout, "rev-parse", "HEAD"]).strip().decode()
 
-        if bootstrapped_vcpkg_version == git_rev:
+        if bootstrapped_vcpkg_version == git_rev_with_patches_applied:
             return 0
 
     print(f"Building vcpkg@{git_rev}")
 
     subprocess.check_call(args=["git", "fetch", "origin"], cwd=vcpkg_checkout)
     subprocess.check_call(args=["git", "checkout", git_rev], cwd=vcpkg_checkout)
 
+    patch_dir = script_dir / "Patches/vcpkg"
```

Comment:
given that this script exists to be windows friendly:
```python
patch_dir = script_dir / "Patches" / "vcpkg"
```
(unless pathlib handles the `/` in the string already)

Reply:
OBE, not using a patch anymore